src/plugins/intel_cpu/tests/functional/subgraph_tests/src/instances/x64/conv_maxpool_activ.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "subgraph_tests/src/classes/conv_maxpool_activ.hpp"
+#include "test_utils/fusing_test_utils.hpp"
+#include "test_utils/filter_cpu_info.hpp"
+#include "ov_models/builders.hpp"
+
+using namespace ngraph;
```

Comment:
Please, don't use `ngraph` at all (exception is only `ngraph::builder` if no replacement in `ov::test::utils`)

Reply:
Done